data-pipeline/src/trace_exporter.rs

```diff
@@ -279,14 +280,20 @@ mod tests {
             exporter.endpoint.url.to_string(),
             "http://127.0.0.1:8126/v0.4/traces"
         );
-        assert_eq!(builder.tracer_version.unwrap(), "v0.1");
-        assert_eq!(builder.language.unwrap(), "nodejs");
-        assert_eq!(builder.language_version.unwrap(), "1.0");
-        assert_eq!(builder.interpreter.unwrap(), "v8");
+        assert_eq!(exporter.tags.tracer_version, "v0.1");
+        assert_eq!(exporter.tags.language, "nodejs");
+        assert_eq!(exporter.tags.language_version, "1.0");
+        assert_eq!(exporter.tags.language_interpreter, "v8");
     }
 
     #[test]
-    fn configure() {}
+    #[ignore]
```

Comment:
This may seem insignificant, but I think this better conveys intention for these empty tests. Otherwise, they always silently pass (reducing the likelihood of never being implemented). Now, cargo will at least print that the tests were ignored. And if you run ignored tests via `cargo test -- --ignored` you'll get a failure saying these tests have not been implemented.